samples/sensor/qdec/README.txt

```diff
@@ -0,0 +1,4 @@
+This sample extends the same-named Zephyr sample to verify it
+with Nordic development kits.
+
+Source code and basic configuration files can be found in the corresponding folder structure in zephyr/samples/sensor/qdec.
```

Comment:
Shouldn't there be a sample documentation for this sample like the other samples?